elm.json

```diff
@@ -9,12 +9,17 @@
             "elm/browser": "1.0.2",
             "elm/core": "1.0.4",
             "elm/html": "1.0.0",
+            "elm/http": "2.0.0",
+            "elm/json": "1.1.3",
             "elm-community/list-extra": "8.2.2",
             "elm-community/maybe-extra": "5.1.0",
-            "mdgriffith/elm-ui": "1.1.5"
+            "krisajenkins/remotedata": "6.0.1",
```

Comment:
You might want to use the specialized [ohanhi/remotedata-http](https://package.elm-lang.org/packages/ohanhi/remotedata-http/latest/)